vllm/engine/arg_utils.py

```diff
@@ -874,7 +881,10 @@ def create_engine_config(self) -> EngineConfig:
             # If not explicitly set, enable chunked prefill by default for
             # long context (> 32K) models. This is to avoid OOM errors in the
             # initial memory profiling phase.
-            if use_long_context:
+
+            # Chunked prefill is currently disabled for multimodal models by
+            # default.
+            if use_long_context and model_config.is_multimodal_model:
```

Comment:
```suggestion
            if use_long_context and not model_config.is_multimodal_model:
```

I think you made a typo here?

Reply:
Since the block under `use_long_context` is what activates chunked prefill.

Reply:
@DarkLight1337 Good catch!